packages-ts/gauntlet-terra/src/commands/internal/terra.ts

```diff
@@ -134,4 +138,21 @@ export default abstract class TerraCommand extends WriteCommand<TransactionRespo
 
     return this.wrapResponse(res)
   }
+
+  async simulate(address, input): Promise<Number> {
+    const msg = new MsgExecuteContract(this.wallet.key.accAddress, address, input)
```

Comment:
Simulate should accept a `signer` param. The simulation should happen using that signer. That will be useful for the multisig case. 
```suggestion
 async simulate(signer: AccAddress, address: AccAddress, input: any): Promise<Number> {
    const msg = new MsgExecuteContract(signer, address, input)
```